workshops/forms.py

```diff
@@ -88,17 +88,8 @@ class BootstrapHelperFormsetInline(BootstrapHelper):
 bootstrap_helper_inline_formsets = BootstrapHelperFormsetInline()
 
 
-class InstructorsForm(forms.Form):
+class WorkshopStaffForm(forms.Form):
     '''Represent instructor matching form.'''
-
-    latitude = forms.FloatField(label='Latitude',
-                                min_value=-90.0,
-                                max_value=90.0,
-                                required=False)
-    longitude = forms.FloatField(label='Longitude',
-                                 min_value=-180.0,
-                                 max_value=180.0,
-                                 required=False)
```

Comment:
That's not the use case I'm thinking of.  Suppose the host institution 
is in Moscow, but we don't have any instructors there. How do I find 
instructors *near* Moscow? The airport probably isn't in our database, 
hence not an option in the pulldown.